mmf/trainers/core/training_loop.py

```diff
@@ -152,3 +144,18 @@ def _extract_loss(self, report: Dict[str, Any]) -> Tensor:
         loss_dict = report.losses
         loss = sum([loss.mean() for loss in loss_dict.values()])
         return loss
+
+    def _calculate_max_updates(self):
+        max_updates = self.training_config.max_updates
+        max_epochs = self.training_config.max_epochs
+        assert max_updates is not None or max_epochs is not None
```

Comment:
We can throw a message here saying both are missing.

Reply:
Done.